lesson_14/homework_14_lms.py

```diff
@@ -3,4 +3,22 @@
 Створіть об'єкт цього класу, представляючи студента.
 Потім додайте метод до класу "Студент", який дозволяє змінювати середній бал студента.
 Виведіть інформацію про студента та змініть його середній бал.
-"""
\ No newline at end of file
+"""
+class Student():
+    def __init__(self, name:str, surname:str, age:int, mark:int):
+        self.name = f"ім'я: {name}"
+        self.surname = f"прізвище: {surname}"
+        self.age = f"вік: {age}"
+        self.mark = f"середній бал: {mark}"
+
+    def change_mark(self, new_mark: int):
+        if new_mark < 0:
+            raise ValueError ("New mark must be positive")
+        if new_mark > 100:
+            raise ValueError ("New mark must less than 100")
+        self.mark = f"середній бал: {new_mark}"
```

Comment:
```suggestion
        self.mark = f"середній бал: {new_mark}"
```
тут пропертя з сетером і етером для відмітки аж проситься, не?